utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/sink/AvroSinkValueParameter.scala

```diff
@@ -79,6 +81,6 @@ object AvroSinkSingleValueParameter {
       isLazyParameter = true,
       defaultValue = defaultValue.map(_.expression)
     )
-    SinkSingleValueParameter(parameter)
+    SinkSingleValueParameter(parameter, new AvroSchemaOutputValidator(schema, ValidationMode.lax))
```

Comment:
why hardcoded lax here? Is it only used in some legacy code where we don't have this validation mode? If so, IMO we should move it upper to those legacy classes